src/odemis/util/test.py

```diff
@@ -108,11 +108,11 @@ def stop_backend():
         raise IOError("Backend failed to stop, now %s" % status)
 
 
-def assert_pos_almost_equal(actual, expected, *args, **kwargs):
+def assert_pos_almost_equal(actual, expected, ignore_mismatch=False, *args, **kwargs):
```

Comment:
It's usually confusing to have parameter/variable names with negation. Let's call this new param something like `match_all=True`. 
Please, also add a docstring about this parameter ("if False, only the expected keys are checked, and actual can have more keys").